README.md

```diff
@@ -1034,6 +1034,75 @@ const MyComponent = (type) => {
 }
 ```
 
+### 7.8 Using spread operator
+
+When creating a component wrapper we can spread the types from our original component. That way the wrapper extends all the props from the original component automatically. Creating a custom type and passing on `React.FC` typescript can give us all the correct props. This is useful to avoid creating a custom interface for our wrapper with missing props from the original component.
+
+**✅ Good:**
+
+```tsx
+import { MenuItem, TextField } from '@mui/material';
+import { TextFieldProps } from '@mui/material';
+
+export type SelectOption = { value: string; label: string };
+
+export type SelectProps = TextFieldProps & {
+  options: SelectOption[];
+};
+
+const Select: React.FC<SelectProps> = ({ options, ...props }) => {
```

Comment:
I think using React.FC to define prop types should not be encouraged. This is a old practice that I don't see anymore in the community (see a [discussion](https://github.com/facebook/create-react-app/pull/8177) in CRA project). 

Instead we could define type direct in prop argument: `MyComponent(props: MyComponentProps)`. Another example:

```suggestion
const Select = ({ options, ...props }: SelectProps) => {
```